File: public/server/lua/src/usr/testListQueue.lua
return require("umd").define({
    "exports",
    "other/utils",
}, function(exports, utils)

    function exports.main(input)

        input.GetSubVal(input.UID, "mList", "") -- get nothing
        input.GetAndLock()

        input.PutSubVal(
            input.UID,
            "mList",
            "1", input.Requ.first,
            "2", input.Requ.second
        ):List(5) -- limit 5
        
        input.PutAndUnlock()
        return {}
    end

end)

File: public/server/go/src/usr/testListQueue.go
package usr

import (
	"GoPluginMagicModule/src/helper"

	"github.com/lockval/go2plugin"
)

func (e export) Export_testListQueue(input *go2plugin.Input) map[string]any {
	input.GetSubVal(input.UID, "mList", "") //get nothing
	input.GetAndLock()

	first := helper.Any2type[string](input.Requ["first"])
	second := helper.Any2type[string](input.Requ["second"])

	input.PutSubVal(
		input.UID,
		"mList",
		"1", first,
		"2", second,
	).List(5) //limit 5
	input.PutAndUnlock()

	return helper.RespEmpty
}
